deployment/migration-assistant-solution/package.json

```diff
@@ -22,8 +22,8 @@
     "@types/eslint__js": "^8.42.3",
     "@types/jest": "^29.5.5",
     "@types/node": "^20.9.0",
-    "aws-cdk": "2.105.0",
-    "aws-cdk-lib": "2.105.0",
+    "aws-cdk": "2.167.2",
```

Comment:
`ipProtocal` was added in a newer version of the CDK, also did the upgrade for it.